exasol/analytics/udf/communication/socket_factory/zmq_wrapper.py

```diff
@@ -149,7 +149,7 @@ def poll(
             self._sockets_map[zmq_socket]: _bitmask_to_flags(bitmask)
             for zmq_socket, bitmask in poll_result.items()
         }
-        return result
+        return result  # type: ignore
```

Comment:
Why do you need the type ignore here?

Reply:
indeed - not needed. Thanks. 
See next push removing the ignore comment.